### PR TITLE
hw-1.2-solution

### PR DESCRIPTION
**Pull request by vbortnyk**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,18 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value <= 1) {
+            return String.valueOf(value);
+        }
+        StringBuilder result = new StringBuilder();
+        while (value > 1) {
+            result.append(String.valueOf(value % 2));
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: it should work without String.valuOf()

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,18 @@
...
+            value /= 2;
+            if (value == 1) {
+                result.append("1");
+                break;
+            }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: can we remove this check?

File: src/main/java/core/basesyntax/BinaryString.java (hunk 1 above)
Comment (vbortnyk) [reply src/main/java/core/basesyntax/BinaryString.java]: done

File: src/main/java/core/basesyntax/BinaryString.java (hunk 2 above)
Comment (vbortnyk) [reply src/main/java/core/basesyntax/BinaryString.java]: done

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/BinaryString.java]: Have you?)

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/BinaryString.java]: I mean if block

Comment (vbortnyk) [reply src/main/java/core/basesyntax/BinaryString.java]: This time it must be correct)

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**